manifest.xml

```diff
@@ -12,7 +12,7 @@
 	<!-- rock library dependencies -->
 	<depend package="gui/vizkit3d" />
 	<depend package="base/types" />
-	
+	<depend package="gui/vizkit" />
```

Comment:
This should definitely not depend on vizkit ... This is a pure vizkit3d plugin. If you have example scripts that use vizkit, add optional="1" to the dependency